kod/object/passive/spell/animate.kod

```diff
@@ -204,6 +206,24 @@ messages:
       return;      
    }
 
+   EndEnchantment(who=$, report=TRUE)
+   {
+      local oMaster;
+
+      oMaster = Send(who, @GetMaster);
+      if report AND IsClass(oMaster, &User)
+      {
+         Send(oMaster, @MsgSendUser, #message_rsc=Animate_expire_rsc , #parm1=Send(who, @GetDef), #parm2=Send(who, @GetName));
+      }
+
+      if Send(who, @GetHealth) > 0
+      {
+         Send(who, @RevertControl);
+         Post(who, @ResetBehaviorFlags);
```

Comment:
This is somewhat unexpected, as it wasn't in the previous implementation in Zombie.  Comment on why it's here?

Reply:
Feel free to correct my interpretation of the logic at any time because I could very well be wrong about it:  My understanding is that, with zombie.kod, RevertControl is never be called if the zombie is killed while affected by animate's control effect since Delete removes the timer.  Since all enchantments are removed when a monster dies (is deleted), this *would* call RevertControl, which is a new behavior for the effect that may or may not be ideal.

I've added a briefer explanation in the code comments.